lib/testLauncher/SuitestLauncher.js

```diff
@@ -68,8 +78,12 @@ class SuitestLauncher {
 							code,
 						}));
 
-					followChildProcess(child, line => {
-						devicesLogs[device.deviceId].push(line);
+					followChildProcess(child, (msg, type) => {
+						ws && ws.write('\n' + msg);
```

Comment:
@andriichumak @givehug should we use platform depended line separators (os.EOL) instead of '\n'?

Reply:
I'm not sure.

Reply:
It probably does not matter at the end, but sure, let's use os.EOL